src/workers/jobs.js

```diff
@@ -184,6 +184,74 @@ export async function processSqsMessages() {
   return p;
 }
 
+export async function completeContactLoad(job, jobMessages) {
+  const campaignId = job.campaign_id;
+  const campaign = await Campaign.get(campaignId);
+  const organization = await Organization.get(campaign.organization_id);
+
+  let deleteOptOutCells;
+  const knexOptOutDeleteResult = await r
+    .knex("campaign_contact")
+    .whereIn("cell", getOptOutSubQuery(campaign.organization_id))
+    .where("campaign_id", campaignId)
+    .delete()
+    .then(result => {
+      deleteOptOutCells = result;
+      console.log("Deleted opt-outs: " + deleteOptOutCells);
+    })
+    .catch(err => {
+      console.log(
+        "Error deleting opt-outs:",
+        campaignId,
+        err
+      );
+    });
+
+  // delete duplicate cells
```

Comment:
Oh nice. Do we currently de-dupe the contacts? I think we do, just checking

Reply:
We don't actually (deduped). We deduped with warehouse loads and it seemed like a good thing to do for all ingest methods